app/design/frontend/Elogic/maksym/Magento_Catalog/templates/product/view/type/default.phtml

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Change text data
+ */
+?>
+<?php /* @var $block \Magento\Catalog\Block\Product\View\AbstractView */?>
+<?php $_product = $block->getProduct() ?>
+
+<?php if ($block->displayProductStockStatus()) :?>
+<?php if ($_product->isAvailable()) :?>
+<div class="stock available" title="<?= $block->escapeHtmlAttr(__('Availability')) ?>">
+    <span><?= $block->escapeHtml(__('Availability: In stock')) ?></span>
+</div>
+<?php else :?>
+<div class="stock unavailable" title="<?= $block->escapeHtmlAttr(__('Availability')) ?>">
+    <span><?= $block->escapeHtml(__('Availability: Out of stock')) ?></span>
+</div>
+<?php endif; ?>
+<?php endif; ?>
```

Comment:
It would be better to use i18n translations for text updates, if you don't need to change html markup

Reply:
noted / done